File: DTO/AddressDto.cs
using System;
using System.Linq;

namespace normalizrTestProject.DbModels.Dto
{
    public class AddressDto
    {
        public Guid Id { get; set; }

        public string Addrline1 { get; set; }

        public string Addrline2 { get; set; }

        public Guid? Personid { get; set; }

        public Guid? Dogid { get; set; }

        public DogDto Dog { get; set; }

        public PersonDto Person { get; set; }
    }
}

File: ClientApp/src/app/models/AddressDto.ts

import { DTOBaseClass } from "./DTOBaseClass";
  
    
import {DogDto} from "./DogDto";

import {PersonDto} from "./PersonDto";



 
export class AddressDto extends DTOBaseClass{ 


public static AddChildProperty_AddressDto_Dog : string= "dog";

public static AddChildProperty_AddressDto_Person : string= "person";



id: string;  

addrline1: string;  

addrline2: string;  

personid: string;  

dogid: string;  

dog: DogDto;  

person: PersonDto;  

}

